src/app/pages/todos/components/todo-card/todo-card.component.html

```diff
@@ -1,7 +1,12 @@
 <form class="form">
   <span class="label">{{ buildTranslationKey('status') | translate }}</span>
 
-  <app-dropdown [formControl]="statusControl" [status]="todo.status"></app-dropdown>
+  <app-dropdown
+    [formControl]="statusControl"
+    [name]="buildTranslationKey(todo.status)"
+    [options]="options"
+    [statusClass]="statusClass">
```

Comment:
now your dropdown component should be generic and shouldn't depend on some status